index.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Mobile</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <!-- header-->
+  <section class="Mobile-Main">
+    <section class="header_sec">
+      <header class="header">
+        <a class="logo">
+          <h1>My Logo</h1>
+        </a>
+        <!-- Navigation Bar-->
+        <nav id="menu">
+          <i class="fa-solid fa-bars dark-blue"></i>
+        </nav>
+      </header>
+      <!--headline section -->
+      <div class="Ellipse">
+
+        <div class="Ellp_one">
+        </div>
+
+        <div class="Rec_one">
+        </div>
+
+        <div class="Ellp_two">
+        </div>
+
+        <div class="Ellp_three">
+        </div>
+        <div class="Ellap_four">
+
+        </div>
+
+      </div>
+      <div class="Info_text">
+        <h2 class="name">I’m Davis Chirao. Nice to meet you!</h2>
```

Comment:
As this is your main heading that's why I recommend you to use `<h1>` tag for the main title of your HTML document and there should be only one `<h1>` tag in an HTML document. hx tags (hx = h2 ... h6) should be use for subtitles.